README.md

```diff
@@ -12,3 +12,37 @@ the function and run automatically when you commit through a GitHub action.
 
 What is the span of the parallel program, in terms of worst-case $\Theta$? Hint:
 It may help to consider the DAG of the parallel program.
+
+## Answer
+We determine the span of the parallel mergesort implementation by analyzing the msort function. 
+1. Recursive Splitting:
+   - The array is split into 2 halves recursively.
+   - The recursion depth is determined by the number of splits needed to reach the base case.
+   - This takes $\log_2(n)$ levels of recursion.
+2. Parallel Recursive Calls:
+   - The left and right halves are sorted concurrently. This does not affect the span.
+3. Merge:
+   - At each of the $\log_2(n)$ levels of recursion, merging all subarrays at that level collectively takes $O(n)$ work, but these merges do not affect the 
+     span.
+
+The total work is similar to the recursive implementation: $O(n \cdot \log_2(n))$.  
+However, the span is determined by the recursive depth, which is $\Theta(\log_2(n))$.
```

Comment:
But are you doing a constant amount of work at each depth?

Reply:
I thought initially that the merge function would contribute.  But then talked myself out of it because it was sequential. So would we say that since that amount of work happens at each depth that it would contribute to the span?

Reply:
In the DAG model we assumed that each node represents a constant amount of work. If that's not the case here, you need to take it into account.